include/exec/exec-all.h

```diff
@@ -469,11 +469,13 @@ struct TranslationBlock {
 #define CF_USE_ICOUNT  0x00020000
 #define CF_INVALID     0x00040000 /* TB is stale. Set with @jmp_lock held */
 #define CF_PARALLEL    0x00080000 /* Generate code for a parallel context */
+#define CF_LOG_INSTR 0x00100000   /* Generate calls to instruction tracing */
 #define CF_CLUSTER_MASK 0xff000000 /* Top 8 bits are cluster ID */
 #define CF_CLUSTER_SHIFT 24
 /* cflags' mask for hashing/comparison */
-#define CF_HASH_MASK   \
-    (CF_COUNT_MASK | CF_LAST_IO | CF_USE_ICOUNT | CF_PARALLEL | CF_CLUSTER_MASK)
+#define CF_HASH_MASK                                                           \
+    (CF_COUNT_MASK | CF_LAST_IO | CF_USE_ICOUNT | CF_PARALLEL |                \
+     CF_CLUSTER_MASK | CF_LOG_INSTR)
```

Comment:
```suggestion
     CF_LOG_INSTR | CF_CLUSTER_MASK)
```
to match the order they're defined in